test/spanner.ts

```diff
@@ -126,12 +127,29 @@ describe('Spanner with mock server', () => {
         const [rows] = await database.run(query);
         assert.strictEqual(rows.length, 3);
         let i = 0;
-        rows.forEach(row => {
+        (rows as Row[]).forEach(row => {
           i++;
-          assert.strictEqual(row[0].name, 'NUM');
-          assert.strictEqual(row[0].value.valueOf(), i);
-          assert.strictEqual(row[1].name, 'NAME');
-          assert.strictEqual(row[1].value.valueOf(), numberToEnglishWord(i));
+          const [numCol, nameCol] = row;
```

Comment:
```suggestion
          const [row1, row2] = row;
```

Reply:
The array `row` contains the columns of the current row, so I don't think we should change the name of these variables to `row1` and `row2`. The iteration over the array of rows is done on [line 130](https://github.com/googleapis/nodejs-spanner/blob/d4590bc8054de38d8436aa9fcb0d74d8c9de5235/test/spanner.ts#L130).